### DISCUSSION TITLE
Extension activation failed after update github copilot

### DISCUSSION THREAD

**Discussion by darthgabriel** [community/community#61369] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hello After update github copilot yesterday i have a problem to login 

`
[ERROR] [default] [2023-07-20T15:35:39.593Z] GitHub Copilot could not connect to server. Extension activation failed: "getaddrinfo ENOTFOUND api.github.com"
[ERROR] [default] [2023-07-20T15:35:43.848Z] GitHub Copilot could not connect to server. Extension activation failed: "unable to get local issuer certificate"
`

`
## Reachability

- github.com: unable to get local issuer certificate
- copilot-proxy.githubusercontent.com: unable to get local issuer certificate
- default.exp-tas.com: unable to get local issuer certificate
`

can you help me what can i do?

**Comment by darthgabriel** [comment, marked as answer]
solved:

1. enter on %USERPROFILE%\.vscode
2. enter on extension/github.copilot-(X)
3. enter on dist
4. make bk of extension.js
5. open extension.js on vscode
6. find two `get rejectUnauthorized()`
7. change each return value for `false` 
8. save
9. restart

good night

**Reply by hk1chfr** [reply]
It still does not work for me?

**Comment by vetemaa** [comment]
Just reinstalling the extension solved the issue in my case.

**Reply by 0xdolan** [reply]
It worked. Just reinstall it. Thanks.

**Comment by HopSoft-Tech** [comment]
this is the error i keep getting [ERROR] [default] [2024-02-03T01:13:17.036Z] Extension activation failed: "getaddrinfo ENOTFOUND api.github.com" in my case. I have reinstalled and it still doesn't work

**Reply by Brendan-Rogers** [reply]
I also get this error, reinstalling does nothing. Would love help!

**Reply by davecheney** [reply]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by bartzalewski** [reply]
I started getting this error today once my iCloud Private Relay became unavailable. It automatically turned off and turned on after 30 minutes, but now I'm getting this error.

**Reply by jiaweing** [reply]
@bartzalewski +1 I kept thinking it was my network issue and I tried reinstalling copilot, restarting my pc, turns out it was the iCloud Private Relay. Just started occurring yesterday for me, worked fine before.

**Comment by vinojhosan** [comment]
1. Do proxy server setup and start it (like px or cntlm)
 2. Close the VS code current instances - all
 3. Reopen the VS code, it works

**Comment by jschrempp** [comment]
Happened to me just now on my Macbook. I turned off Private Relay and now CoPilot works.
